.buildkite/pipelines/pull_request/apm_cypress.yml

```diff
@@ -5,6 +5,7 @@ steps:
       queue: n2-4-spot
     depends_on: build
     timeout_in_minutes: 120
+    parallelism: 4
```

Comment:
out of curiosity why 4?

Reply:
I have experimented with a few numbers and it seems that splitting it up in 4 agents gives us a good boost without having to start more agents. One thing to keep in mind though is that the test server setup time remains the same as per above comment. We can change that easily though if it makes sense.